### PR TITLE
fix: addressed xbrowser issues (CMC-1693)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1693

### Change description ###
Disables tests in microsoft edge to solve 1.1

Fixes the following issues from [confluence](https://tools.hmcts.net/confluence/pages/viewpage.action?spaceKey=CU&title=Cross+Browser+Tests+Issues&__ncforminfo=nWl6zt-axg0DGnQVIPZkO6yEUcA9FpdwBJKSNjGdKwUGZalLHpMvkwX0IpnPw4bZNs3K1zeN3MvKnIrNOix2YQ==)
        2.1 (also fixes 6.1, 7.1)
        2.4 (also fixes 3.3, 5.2)
        3.1
        3.2 (also fixes 4.1, 5.1, 7.2)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by sabahirfan — DISMISSED** [review]

**Review by adamjhc — DISMISSED** [review]

File: e2e/tests/crossBrowser_test.js
Comment (swalker125) [reply e2e/tests/crossBrowser_test.js:30]: thanks, need to add navigateToCaseDetails method here

File: e2e/crossbrowser/supportedBrowsers.js
Comment (swalker125) [reply e2e/crossbrowser/supportedBrowsers.js:15]: created here https://tools.hmcts.net/jira/browse/CMC-1698

**Review by pd-dudkiewicz — APPROVED** [review]
LGTM :+1: can we just verify if fixes the issues on nightly-dev before merging?

**Comment by swalker125** [comment]
- firefox-firefox_win_latest
[1.safari:firefox]   ✔ Full end-to-end journey in 1194060ms

https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_Nightly_Civil%2Fcivil-service/detail/nightly-dev/127/pipeline

**Review by douglasrice — APPROVED** [review]

**Pull request merged**
